### PR TITLE
added Wikipedia links

### PR DESCRIPTION
**Pull request by Padmanabh82**
## Related Issue

#164 

Fixes: #[information links will be added]

### CODE REVIEW

**Comment by Amit366** [comment]
@Padmanabh82 can u add a screen shot of the page

**Comment by Padmanabh82** [comment]
@Amit366 sorry sir but I have deleted from my computer, can you see from code ?

**Comment by Padmanabh82** [comment]
@Amit366 sir please add this repo today 🙏🙏. Tomorrow hactober is ending

**Comment by Amit366** [comment]
@Padmanabh82 I could see u just changes React Fragment to Fragments. U haven't added the Wikipedia link for each calculator

**Comment by Amit366** [comment]
U needed to add Wikipedia link for each calculator topic

**Comment by Padmanabh82** [comment]
@Amit366 sir see this file `funwithphysics/src/Components/PhysicsTopic/index.js`

**Comment by Amit366** [comment]
@Padmanabh82 I saw this but Wikipedia links should be added under each calculator and not to index.js page

**Comment by Padmanabh82** [comment]
Ok sir

**Comment by Padmanabh82** [comment]
@Amit366 sir but can you give me path of files where should I put links ??

**Comment by Amit366** [comment]
@Padmanabh82 see you there are many paths as there are already many calculators present. So it's better if u can checkout by exploring the codebase.

**Comment by Amit366** [comment]
@Padmanabh82 u can check topics.js

**Comment by Padmanabh82** [comment]
@Amit366 there are folders name of topic not one js file sir

**Comment by Padmanabh82** [comment]
And I got one calculator on web but not got file

**Comment by Padmanabh82** [comment]
@Amit366 sir i am getting this error:-

`Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. You likely forgot to export your component from the file it's defined in, or you might have mixed up default and named imports.

Check the render method of `Calculator`.
▶ 22 stack frames were collapsed.
Module.<anonymous>
src/index.js:9
   6 | import {BrowserRouter} from "react-router-dom";
   7 | 
   8 | 
>  9 | ReactDOM.render(
  10 |   <React.StrictMode>
  11 |     <BrowserRouter>
  12 |     <App />
View compiled
Module../src/index.js
http://localhost:3000/static/js/main.chunk.js:14220:30
__webpack_require__
/home/pi/FunwithPhysics/funwithphysics/webpack/bootstrap:856
  853 | 
  854 | __webpack_require__.$Refresh$.init();
  855 | try {
> 856 | 	modules[moduleId].call(module.exports, module, module.exports, hotCreateRequire(moduleId));
      | ^  857 | } finally {
  858 | 	__webpack_require__.$Refresh$.cleanup(moduleId);
  859 | }
View compiled
fn
/home/pi/FunwithPhysics/funwithphysics/webpack/bootstrap:150
  147 | 		);
  148 | 		hotCurrentParents = [];
  149 | 	}
> 150 | 	return __webpack_require__(request);
      | ^  151 | };
  152 | var ObjectFactory = function ObjectFactory(name) {
  153 | 	return {
View compiled
1
http://localhost:3000/static/js/main.chunk.js:14343:18
__webpack_require__
/home/pi/FunwithPhysics/funwithphysics/webpack/bootstrap:856
  853 | 
  854 | __webpack_require__.$Refresh$.init();
  855 | try {
> 856 | 	modules[moduleId].call(module.exports, module, module.exports, hotCreateRequire(moduleId));
      | ^  857 | } finally {
  858 | 	__webpack_require__.$Refresh$.cleanup(moduleId);
  859 | }
View compiled
checkDeferredModules
/home/pi/FunwithPhysics/funwithphysics/webpack/bootstrap:45
  42 | 	}
  43 | 	if(fulfilled) {
  44 | 		deferredModules.splice(i--, 1);
> 45 | 		result = __webpack_require__(__webpack_require__.s = deferredModule[0]);
     | ^  46 | 	}
  47 | }
  48 | 
View compiled
Array.webpackJsonpCallback [as push]
/home/pi/FunwithPhysics/funwithphysics/webpack/bootstrap:32
  29 | 	deferredModules.push.apply(deferredModules, executeModules || []);
  30 | 
  31 | 	// run deferred modules when all chunks ready
> 32 | 	return checkDeferredModules();
     | ^  33 | };
  34 | function checkDeferredModules() {
  35 | 	var result;
View compiled
(anonymous function)
http://localhost:3000/static/js/main.chunk.js:1:79
This screen is visible only in development. It will not appear if the app crashes in production.
Open your browser’s developer console to further inspect this error.  Click the 'X' or hit ESC to dismiss this message.`

after clicking on every calculator of classical mechanics and i have done it for thermodynamics calculator i will comit thah but please accpt my pr

**Comment by Padmanabh82** [comment]
@Amit366 your new merged code dont work in classical mechanics

**Comment by Amit366** [comment]
@Padmanabh82 it's working fine on others PC and I haven't pushed any code now so check it out. Else wait till 8 pm

**Comment by Padmanabh82** [comment]
@Amit366 see sir your commit has failed the test

**Comment by Padmanabh82** [comment]
And sir I have not removed much lines only react fragments and that also on other file

**Comment by Amit366** [comment]
@Padmanabh82 see there is no error and also others are not facing any problem.

**Comment by Amit366** [comment]
@Padmanabh82 before u sent the PR didn't u get any error and did u run it on ur PC??

**Comment by Padmanabh82** [comment]
@Amit366 sir error comes only when I try to load any calculator of Mechanics

**Comment by Amit366** [comment]
@Padmanabh82 did u install node modules??

**Comment by Padmanabh82** [comment]
@Amit366 si how it will work without it

**Comment by Amit366** [comment]
@Padmanabh82 it's working fine for others

**Comment by Padmanabh82** [comment]
@Amit366 I have done it

**Comment by Amit366** [comment]
Anyways let me check

**Comment by Amit366** [comment]
> @Amit366 I have done it

What?

**Comment by Padmanabh82** [comment]
Ok

**Comment by Padmanabh82** [comment]
> > @Amit366 I have done it
> 
> What? @Padmanabh82

@Amit366 installing node modules

**Comment by Amit366** [comment]
@Padmanabh82 install the node modules inside the fun with physics folder

**Comment by Padmanabh82** [comment]
@Amit366 sir I know it

**Comment by Padmanabh82** [comment]
Done

**Comment by Padmanabh82** [comment]
@Amit366 Please add my pr sir

**Comment by Amit366** [comment]
@Padmanabh82 did it work now??

**Comment by Padmanabh82** [comment]
@Amit366 sir it was installed before I was developing

**Comment by Padmanabh82** [comment]
@Amit366 sir problem is not made by me as I send screenshot to you and Please accept my pr

**Comment by Amit366** [comment]
@Padmanabh82 see I deleted the project from my PC and then re cloned it but it is working fine. And for the sake of Hacktober fest I cannot accept the half made PR. So maybe the problem is from ur side as I re cloned it but it's working fine.

**Comment by Padmanabh82** [comment]
Mechanics calculators were also working ??

**Comment by Amit366** [comment]
@Padmanabh82 yes

**Comment by Amit366** [comment]
@Padmanabh82 it's still incomplete as u haven't added to other calculators so I am closing this PR.

File: funwithphysics/src/Components/Classical Mechanics/wpe_data.js [hunk 1]
```diff
@@ -14,16 +14,17 @@ const Topics = [
...
+        details: `Work is said to be done when a body or object moves with the application of external force. We can define work as an activity involving a movement and force in the direction of the force. For example, a force of 20 newtons (N) pushing an object 3 meters in the same direction of the force will do 60 joules (J) of work .For more information visit ${<a href="https://en.wikipedia.org/wiki/Work_(physics)" target="_blank" rel="noreferrer">Wikipedia</a>}`,
         formula: "Work = Force × Displacement = F × d",
         process: "When we kick a football, we are exerting an external force called F and due to this force (kick), the ball moves to a certain distance. This disposition of ball from position A to B is known as displacement (d). This work is said to be done and can be calculated as W = F × d ",
         siunit: "Joule or Newton-Meter",
-        dimension: "ML²/T²"
+        dimension: "ML²/T²",
+        wikilink:"https://en.wikipedia.org/wiki/Work_(physics)"
```
Comment (Amit366) [inline funwithphysics/src/Components/Classical Mechanics/wpe_data.js:22]: Why have u added wikilink separately??

File: funwithphysics/src/Components/PhysicsTopic/index.js [hunk 2]
```diff
@@ -42,35 +42,35 @@ function PhysicsTopic() {
...
   							</PhysicsTopicItem>
 						<PhysicsTopicItem name="Relativity" link="/relativity" icon="icon-lab2" image={relativity}>
-							<p>Relativity is the notion that the laws of physics are the same everywhere, be it on Earth or in a far off corner of the universe.</p>
+							<p>Relativity is the notion that the laws of physics are the same everywhere, be it on Earth or in a far off corner of the universe. For more Information visit <a href="https://en.wikipedia.org/wiki/Theory_of_relativity" target="_blank" rel="noreferrer">Wikipedia</a></p>
 						    <p id="credit">Image by <a href="https://pixabay.com/users/stux-12364/?utm_source=link-attribution&amp;utm_medium=referral&amp;utm_campaign=image&amp;utm_content=4129246">Thanks for your Like • donations welcome</a> from <a href="https://pixabay.com/?utm_source=link-attribution&amp;utm_medium=referral&amp;utm_campaign=image&amp;utm_content=4129246">Pixabay</a></p>
```
Comment (Amit366) [inline funwithphysics/src/Components/PhysicsTopic/index.js:54]: Remove the Wikipedia link from this page.

File: funwithphysics/src/Components/Thermodynamics/Topics/Calculator.js [hunk 3]
```diff
@@ -226,6 +226,7 @@ function Calculator({ match }) {
       </div>
       <div className="Calculator__details">
         <p>{details.details}</p>
+        For more Information visit <a href="https://en.wikipedia.org/wiki/Kinetic_theory_of_gases" target="_blank" rel="noreferrer">Wikipedia</a>
       </div>
```
Comment (Amit366) [inline funwithphysics/src/Components/Thermodynamics/Topics/Calculator.js:230]: U have already added the link along with the topic details then why u again added here?

**Comment by Padmanabh82** [comment]
@Amit366 see it i have added all links

File: funwithphysics/src/Components/Home/Home.js [hunk 4]
```diff
@@ -25,12 +25,12 @@ const Home = () => {
 			<Footer />
 			{/* <!-- Back to top button --> */}
 			<button className="gotopbtn" onClick={scroll}> <i className="fas fa-arrow-up"></i> </button>
-		</React.Fragment>
+		</>
```
Comment (Amit366) [inline funwithphysics/src/Components/Home/Home.js:28]: Why did u remove React Fragments?

**Comment by Amit366** [comment]
@Padmanabh82 u haven't added the link for error measurement

File: funwithphysics/src/Components/Classical Mechanics/wpe_data.js (hunk 1 above)
Comment (Padmanabh82) [reply funwithphysics/src/Components/Classical Mechanics/wpe_data.js:22]: wait its old idea i will remove it

**Comment by Padmanabh82** [comment]
> @Padmanabh82 u haven't added the link for error measurement

@Amit366 sir i didt got link for it

File: funwithphysics/src/Components/Home/Home.js (hunk 4 above)
Comment (Padmanabh82) [reply funwithphysics/src/Components/Home/Home.js:28]: </> this also works thatswhy

File: funwithphysics/src/Components/Thermodynamics/Topics/Calculator.js (hunk 3 above)
Comment (Padmanabh82) [reply funwithphysics/src/Components/Thermodynamics/Topics/Calculator.js:230]: ok i will remove it

File: funwithphysics/src/Components/PhysicsTopic/index.js (hunk 2 above)
Comment (Padmanabh82) [reply funwithphysics/src/Components/PhysicsTopic/index.js:54]: ok

File: funwithphysics/src/Components/Classical Mechanics/wpe_data.js (hunk 1 above)
Comment (Amit366) [reply funwithphysics/src/Components/Classical Mechanics/wpe_data.js:22]: @Padmanabh82 u haven't changed it why have u marked as resolved.

File: funwithphysics/src/Components/Home/Home.js (hunk 4 above)
Comment (Amit366) [reply funwithphysics/src/Components/Home/Home.js:28]: See first of all u said it's not working on ur PC and then when u r changing something how would u know whether it is working or not?

**Comment by Amit366** [comment]
@Padmanabh82 don't mark as resolved

File: funwithphysics/src/Components/Classical Mechanics/wpe_data.js (hunk 1 above)
Comment (Padmanabh82) [reply funwithphysics/src/Components/Classical Mechanics/wpe_data.js]: done

File: funwithphysics/src/Components/Home/Home.js (hunk 4 above)
Comment (Padmanabh82) [reply funwithphysics/src/Components/Home/Home.js:28]: sir close this

File: funwithphysics/src/Components/Thermodynamics/Topics/Calculator.js (hunk 3 above)
Comment (Padmanabh82) [reply funwithphysics/src/Components/Thermodynamics/Topics/Calculator.js]: removed !!

File: funwithphysics/src/Components/PhysicsTopic/index.js (hunk 2 above)
Comment (Padmanabh82) [reply funwithphysics/src/Components/PhysicsTopic/index.js:54]: done

**Comment by Padmanabh82** [comment]
@Amit366 is it ok now ??

**Comment by Amit366** [comment]
@Padmanabh82 I have to close this as it there is merge conflict which hasn't been resolved yet.

**Pull request closed without merging**